### ISSUE TITLE
[flutter_gpiod] GPIO Event not work & exception occurred 

### ISSUE THREAD

**Issue by JAICHANGPARK** [ardera/flutter_packages#8]
Hi I using flutter gpiod 

but my rpi4 is not work gpio event code 
how to solved this problem? 

```
Unhandle exception:
Could not wait for GPIO events.. epoll_wait: Interrupted system call
```

**Comment by ardera** [comment]
Ah my bad. This is fixed in the repo already, but I forgot to publish a new version.

**Comment by JAICHANGPARK** [comment]
@ardera 

Thank you for making such a great package !

**Comment by ardera** [comment]
thx! I'll close this once I've uploaded the new version

**Comment by ardera** [comment]
- fixed in 2ab2eb939c491a559d267cb380d75a068e10f7bc
